src/Movies/Movies.js

```diff
@@ -0,0 +1,61 @@
+
+import React, { Component } from 'react'
+import axios from 'axios'
+import "./movies.css"
+export default class Movies extends Component {
+    constructor(){
+        super()
+        this.state={
+            search: '',
+            movies: {},
```

Comment:
Since state.movies stores an object describing a single movie, I would recommend naming it `movie` without the 's'